mne/externals/__init__.py

```diff
@@ -3,3 +3,4 @@
 from . import decorator
 from . import tempita
 from . import h5io
+from . import pacpy
```

Comment:
ah removing that fixed it

On Tue, Jan 12, 2016 at 12:59 PM, Eric Larson <notifications@github.com>
wrote:

> In mne/externals/__init__.py
> <https://github.com/mne-tools/mne-python/pull/2306#discussion_r49512483>:
>
> > @@ -3,3 +3,4 @@
> >  from . import decorator
> >  from . import tempita
> >  from . import h5io
> > +from . import pacpy
>
> Did you get rid of this line when you tried nesting all pacpy imports?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2306/files#r49512483>.
>